### DISCUSSION TITLE
NTS on npgsql

### DISCUSSION THREAD

**Discussion by abo-joroma** [NetTopologySuite/NetTopologySuite#639] [Q&A]
Greetings everyone. I would like to ask a simple question. And that is, whether I need to install the PostGIS extension on my Postgres instance before I can start using NTS. 

If not, please let me know.

**Comment by DGuidi** [comment]
please post this on [PostGIS](https://github.com/NetTopologySuite/NetTopologySuite.IO.PostGis), but yes. If you plan to use the PostGIS Reader/Writer you need PostGIS extensions.
